stories/pages/Curriculum.stories.tsx

```diff
@@ -54,27 +54,36 @@ export const WithAlerts: React.FC = () => {
   )
 }
 
-export const CompletedLessons: React.FC = () => {
-  const session = {
-    ...dummySessionData,
+export const CompletedLessons: React.FC<{}> = () => {
+  const session: Session = {
```

Comment:
It's actually quite funny. We used `dummySessionData` data in so many tests but its types are incorrect. `id` filed needs to be a string.